src/lib.rs

```diff
@@ -109,6 +108,9 @@ pub mod jellyfish {
         pub const BLSSIGNINGKEY: &str = "BLSSIGNINGKEY";
         pub const BLSSIG: &str = "BLSSIG";
 
+        pub const SCHNORRSIGNKEYPAIR: &str = "SCHNORRSIGNKEYPAIR";
         pub const SCHNORRVERKEY: &str = "SCHNORRVERKEY";
+        pub const SCHNORRSIGNINGKEY: &str = "SCHNORRSIGNINGKEY";
+        pub const SCHNORRSIG: &str = "SCHNORRSIG";
```

Comment:
can we use snake case, this is hard to read.

Reply:
Since we're already making backwards incompatible changes, yes why not!

Reply:
verygoodidea

Reply:
sry, this is really minor, but I think we should always delimit between two words, such as `SCHNORR_SIGNING_KEY` instead of `SCHNORR_SIGNINGKEY`

Reply:
@alxiong Since we're at it, why don't we rename to the notation used in [the draft](https://datatracker.ietf.org/doc/draft-irtf-cfrg-bls-signature/)? Some [libs](https://github.com/arkworks-rs/crypto-primitives/blob/bd447455510878adf82894cb23766dc914c31c32/src/signature/mod.rs) seem to follow the same notation, while [others](https://github.com/RustCrypto/signatures) use mixed names (e.g. SecretKey -> SigningKey).
That would, of course, mean that we'd need to rename the types on our [`SignatureScheme`](https://github.com/EspressoSystems/jellyfish/blob/9bc08d2e5e943ea2610eb02064263ceb38478280/primitives/src/signatures/mod.rs#L15) trait in jellyfish.

Reply:
Generally, I don't think we need to follow IRTF draft for naming convention. 

But on the topic of "SigningKey" v.s. `SecretKey`, I don't have a preference. [Boneh's book](https://toc.cryptobook.us/book.pdf#chapter.13) says "signing", [wiki ](https://en.wikipedia.org/wiki/Digital_signature)and many standard use the more general "secret".

under our current situation, I'd prefer keeping the associated type names in `trait SignatureScheme` unchanged.